src/Mod/Path/PathScripts/PathPost.py

```diff
@@ -291,7 +291,7 @@ def Activated(self):
                 # create an object to serve as the fixture path
                 fobj = _TempObject()
                 c1 = Path.Command(f)
-                c2 = Path.Command("G0 Z" + str(job.Stock.Shape.BoundBox.ZMax))
+                c2 = Path.Command("G0 Z" + str(job.Stock.Shape.BoundBox.ZMax + job.SetupSheet.ClearanceHeightOffset.Value))
```

Comment:
This is probably not very transparent for the user - especially because they set the ClearanceHeight directly in the op (and might've never used the setup sheet).

I looked a bit around and couldn't find an obvious way of how to make this transparent though. The only thing I could come up with is to add a `ClearanceHeight` value to the Job and show it in the output section with the explanation that it is used for specifically this purpose. Default value could be above calculation as an expression.

Just a thought.

Reply:
The linked forum post is from users confused because they don't understand where these moves are coming from so anything that can improve that is a benefit. 

What about adding a comment with each 'c2' command explaining what it is for?

"Rapid to clearance height defined in the job setup sheet "